src/qt/bitcoinstrings.cpp

```diff
@@ -17,7 +17,8 @@ QT_TRANSLATE_NOOP("bitcoin-core", ""
 "If the file does not exist, create it with owner-readable-only file "
 "permissions.\n"
 "It is also recommended to set alertnotify so you are notified of problems;\n"
-"for example: alertnotify=echo %%s | mail -s \"Peercoin Alert\" admin@foo.com\n"),
+"for example: alertnotify=echo %%s | mail -s \"Peercoin Alert\" admin@foo."
+"com\n"),
```

Comment:
Is this correct?

Reply:
It possibly is an accidental return.
But it doesn't impact compilation, it's just syntactically different.
Good catch though, will revert this later.